crates/node/src/cli/mod.rs

```diff
@@ -202,23 +201,6 @@ pub enum NetworkType {
 }
 
 impl NetworkType {
-    pub fn uri(&self) -> &'static str {
-        match self {
-            NetworkType::Main => "https://alpha-mainnet.starknet.io",
-            NetworkType::Test => "https://alpha-sepolia.starknet.io",
-            NetworkType::Integration => "https://integration-sepolia.starknet.io",
-            NetworkType::Devnet => unreachable!("Gateway url isn't needed for a devnet sequencer"),
-        }
-    }
-
-    pub fn gateway(&self) -> Url {
-        Url::parse(&format!("{}/gateway/", self.uri())).expect("Invalid uri")
-    }
-
-    pub fn feeder_gateway(&self) -> Url {
-        Url::parse(&format!("{}/feeder_gateway/", self.uri())).expect("Invalid uri")
-    }
-
     pub fn chain_id(&self) -> ChainId {
```

Comment:
why keep this?

Reply:
This is still in use [here](https://github.com/madara-alliance/madara/blob/d7eb7bf00c412a52bb33dcb1d9c4a0d90f05866f/crates/node/src/main.rs#L197)